libc/src/stdbit/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+add_entrypoint_object(
+  stdc_leading_zeros_uc
+  SRCS
+    stdc_leading_zeros_uc.cpp
+  HDRS
+    stdc_leading_zeros_uc.h
+)
```

Comment:
Can you add `DEPENDS libc.src.__support.CPP.bit` to these entrypoints?

Reply:
fixed in 7a898f3330e5, good catch (I'll get the hang of this)